Documentation/Getting started with IntelliSense configuration.md

```diff
@@ -1,17 +1,3 @@
-## How to build and debug the Microsoft CppTools Extension
-
```

Comment:
I think this version of the file should be used. It looks like the new file has the old contents?

Reply:
I didn't touch the content of the file...which link isn't working?

Reply:
The contents were changed in master. The code you deleted doesn't match the code you added. The last link doesn't work.

Reply:
e.g. The `** **` are missing.

Reply:
ah, didn't realize the content has been modified by someone else in master. i'll update.